libs/util/shared/src/lib/image-fallback.directive.ts

```diff
@@ -7,7 +7,13 @@ export class ImageFallbackDirective {
   @Input() fallbackUrl: string
   @HostListener('error')
   useFallback() {
-    this.el.nativeElement.src = this.fallbackUrl
+    const fallbackUrl = new URL(
+      this.fallbackUrl,
+      window.location.toString()
+    ).toString()
+    if (this.el.nativeElement.src !== fallbackUrl) {
+      this.el.nativeElement.src = fallbackUrl
+    }
```

Comment:
Just for my understanding, why do we need to add this logic?

Reply:
if the fallback url gives an error (e.g. 404), then the directive is stuck in an infinite loop because it tries to set the fallback url again and again